CODING_STANDARDS

```diff
@@ -250,8 +250,7 @@ Syntax and indentation
 Testing
 -------
 
-1.  Extensions should be well tested using *.phpt tests. Read about that
-    in README.TESTING.
+1.  Extensions should be well tested using *.phpt tests.
```

Comment:
Shouldn't we add a link http://qa.php.net/write-test.php?

Reply:
Absolutely.. Added in the next commit.